src/main/java/org/mastodon/views/trackscheme/display/TrackSchemePanel.java

```diff
@@ -197,17 +199,18 @@ public TrackSchemePanel(
 			final TrackSchemeGraph< ?, ? > graph,
 			final HighlightModel< TrackSchemeVertex, TrackSchemeEdge > highlight,
 			final FocusModel< TrackSchemeVertex, TrackSchemeEdge > focus,
-			final TimepointModel timepoint,
+			final TimepointModel navigationTimepointModel, final TimepointModel fadingTimepointModel,
```

Comment:
Please add a new line after the comma, to keep the rule of one parameter per line.

Reply:
Done (in the new implementation with the FadedModel).